docs/cheatsheet.rst

```diff
@@ -139,40 +139,50 @@ The steps are as follows:
 
 This is where the actual magic happens.
 
-4. Commit it
+4. Add it
+
+-   add a single file
+::
+    hub add  filename
+
+-   add all changed files
+::
+    hub add  .
+
+5. Commit it
 ::
 
     hub commit -a -m "hey look, real work!"
 
-5. Push it back up to github
+6. Push it back up to github
```

Comment:
I think we should change this to a numbered list, using `#` instead of the actual numbers. This will save future editors from having to do this :)
https://thomas-cokelaer.info/tutorials/sphinx/rest_syntax.html#list-and-bullets